11-if/exercises/03-arg-count/main.go

```diff
@@ -30,4 +25,15 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	l := len(os.Args) - 1
```

Comment:
same as earlier comment; name it appropriately like num or count or c

Reply:
Fixed